style.css

```diff
@@ -1,136 +1,94 @@
 }
 
-li {
-    display: inline;
-    padding-bottom: 200px;
-    
+body {
+    background-color: black;
 }
 
+ul li {
+    display: inline-block;
+}
 
-
-header {
-    display: flex;
-    width: 1002.34;
-    height: 100%;
-    padding-left: 75px;
-    padding-right: 75px;
-    justify-content: space-between;
- 
-
-
-    
+ul {
+    display: inline-block;
 }
 
-nav {
+li a {
     text-decoration: none;
-    list-style: none;
-    display: flex;
-    justify-content: space-between;
-    line-height: 30px;
-    
-   
-    
-
-
+    color: #4D4D4D;
+    padding: 16.71px;
 }
-.buttom{
-    background-color: #4CAF50;
-    border-radius: 3px;
+
+.button {
+    background-color: #4CAF4F;
     color: white;
-    padding: 12px 20px;
-    text-align: center;
-    text-decoration: none;
-    display: inline-block;
-    font-size: 11.14;
-    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    border-radius: 2.78px;
 }
 
-nav a{
-    padding: 5px;
-    line-height: 13px;
-
-
-   
-}
-.buttom a{
-    vertical-align: 20px;
+.RightArrow {
+    vertical-align: baseline;
+    height: 11.14px;
+    width: 11.14px;
 }
-.les{
+
+a,
+h1,
+h2,
+p {
+    font-family: 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
     color: #4D4D4D;
-    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
 }
-#from{
-    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+
+h2 span {
     color: #4CAF4F;
+    font-size: 44.55px;
 }
-.greytext{
-    color: #4D4D4D;
-    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
-    font-size: 11px;
-}
-#reg{
-    background-color: #4CAF50;
-    border-radius: 3px;
-    color: white;
-    padding: 17px 22px;
-    text-align: center;
-    text-decoration: none;
-    display: inline-block;
-    font-size: 11.14;
-    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+
+.block1 h2 {
+    font-size: 44.55px;
 }
-#block1{
-    display: inline-block;
-    
-    
+
+.Subtext {
+    color: #717171;
+    font-size: 11.14px;
 }
-#man{
-    width: 272px;
-    height: 283px;
-    float: right;
+
+.block1 {
+    background-color: #F5F7FA;
+    justify-content: space-between;
+    display: flex;
 }
-#allBlock1 {
-    margin: 67px 100px 67px 100px;
-    
 
+.imgBlock1 {
+    width: 272.16px;
+    height: 283.3px;
 }
-#dot{
-    text-align:center;
-    margin-bottom: 28px;
+.dots{
+    background-color: #F5F7FA;
+    text-align: center;
 }
-#OurClients{
+.clients {
+    background-color: white;
+    display: block;
     text-align: center;
-    padding: 0%;
-    margin-bottom: 28px;
 }
-#logos{
-    justify-content: space-around;
-    display: flex;
-    margin: 11px 100px 0px 100px;
+.Titletext{
```

Comment:
Decide one way to name classes. You have classes from small letters, here you have from camel case letter